plugins/samson_ledger/test/lib/samson_ledger/samson_plugin_test.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+require_relative "../../test_helper"
+
+SingleCov.covered! file: 'plugins/samson_ledger/lib/samson_ledger/samson_plugin.rb'
+
+describe "the plugin callback" do
```

Comment:
when multiple tests fail this description will not be helpful ... nest under `describe SamsonLedger do`